frontend/src/components/common/form/ActionButton.tsx

```diff
@@ -84,7 +89,12 @@ const ActionButton = <Default extends boolean = true>({
           : generateErrorMessage) ??
         "An error occurred. Please try again later.";
       const errorMessage =
-        e instanceof FormValidationError ? e.message : nonValidationMessage;
+        e instanceof ApolloError &&
+        e.message == "Response not successful: Received status code 413"
```

Comment:
not the cleanest, but `e` doesn't have a status code field - only `message` is useful

Reply:
move this into server.ts as wrapper for `graphqlUploadExpress`?